### DISCUSSION TITLE
Cannot get podman 4.x on RHEL 8

### DISCUSSION THREAD

**Discussion by imperialguy** [containers/podman#13799] [General]
Despite several attempts to upgrade/update container-tools in RHEL8, the latest version it shows is still podman 3.4.2. How can I get the latest podman 4.x release on RHEL 8?

**Comment by Luap99** [comment]
Podman 4.0 will land in RHEL 8.6

**Reply by imperialguy** [reply]
Okay, then how do I at least upgrade to podman 3.4.4 on RHEL 8.5? The following doesn't do anything:

`dnf -y update podman`
`dnf -y module update container-tools`

**Reply by Luap99** [reply]
RHEL only uses specific versions. I think the version is only updated for point releases, after that only approved bug fixes are backported.

It would be better if you reach out to Red Hat support and not us (upstream) and ask them about this.
